certified-connectors/InEight/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "InEight_ClientId",
```

Comment:
@InEightDevOps You can update it to `"[[DUMY]"` or `"CLIENTID"` to maintain consistency with other connector submissions.